### PR TITLE
Fix project creation to directly create frontend state

### PR DESCRIPTION
**Pull request by ChrisP19**
- [x] Add import to project creation

- [x] Correctly import and export attributes

- [x] Add tests for import then export

- [ ] Add tests for basic utilities and file storage- started in this PR, but can finish in another because this is blocking

### CODE REVIEW

File: app/src/js/server/export.ts [hunk 1]
```diff
@@ -33,20 +33,25 @@ export function convertItemToExport (config: ConfigType,
...
-      case ShapeType.CUBE:
-        const poly3d = indexedShape.shape as CubeType
-        labelExport.box3d = poly3d
-        break
+    // TODO: fix this to loop over all labels shapes
+    // right now it just exports the first one
+    if (label.shapes.length > 0) {
+      const shapeId = label.shapes[0]
+      const indexedShape = item.shapes[shapeId]
+      switch (indexedShape.type) {
```
Comment (gycn) [inline app/src/js/server/export.ts]: This should actually check the label type. There will always be one export per label

File: app/src/js/server/import.ts [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    }
+  })
+  return labelAttributes
+}
+
+ /**
+  * based on the export format label,
+  * get label type, shape type, and shape data
+  */
+function getLabelByType (
```
Comment (gycn) [inline app/src/js/server/import.ts]: I think this function would be better if it returned a complete label, with all the shape fields filled out. This way, if we need to do more complicated initialization in the future, we won't have to change as many places in the code

Comment (ChrisP19) [reply app/src/js/server/import.ts]: Changed to your suggestion

File: app/src/js/server/export.ts (hunk 1 above)
Comment (ChrisP19) [reply app/src/js/server/export.ts]: Changed to check label type

**Pull request merged**